### DISCUSSION TITLE
Every day I get closer to true vanilla world gen in minestom

### DISCUSSION THREAD

**Discussion by Flamgop** [Minestom/Minestom#682] [Showcase]
![2022-02-15_00 35 46](https://user-images.githubusercontent.com/41309430/154024989-f3dc017c-0cbc-4068-90ab-7f013a9f81b4.png)

**Comment by LazyGamer1111** [comment]
nice, looks great.
How long have you been working on it?

**Reply by Flopgop** [reply]
for fucking forever lmao

**Reply by LazyGamer1111** [reply]
> for fucking forever lmao

Can I download the minestrone code for this generation somewhere?

**Reply by Flopgop** [reply]
> > for fucking forever lmao
> 
> Can I download the minestrone code for this generation somewhere?

I don't think I have it anymore unfortunately

**Reply by LazyGamer1111** [reply]
> > > for fucking forever lmao
> 
> > 
> 
> > Can I download the minestrone code for this generation somewhere?
> 
> 
> 
> I don't think I have it anymore unfortunately
> 
> 

Well shit

**Comment by ch99q** [comment]
It may be easier to implement their world generator for their data packs and then use the vanilla data pack for it. That would also allow for using custom world generator data packs like Terralith.

**Reply by Flopgop** [reply]
I don't understand how minecraft's noises work enough to replicate them with the same parameters that are required by the datapacks
If anyone else does though I welcome them to figure it out! (please it's so hard)
